drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -302,34 +294,22 @@ def save_new_enrollment!
   end
 
   def save_in_progress!
-    # "WIP" Enrollments have a null ProjectID column, so that they don't
-    # get pulled into reporting. The Project database ID is stored in the hmis_wips table.
-    saved_project_id = project.id
+    raise 'cannot unset ProjectID on enrollment that is missing project_pk' unless project_pk
 
+    # "WIP" Enrollments have a null ProjectID column, so that they aren't included in reporting.
     self.project_id = nil
-    self.wip = Hmis::Wip.find_or_create_by(
-      {
-        source: self,
-        project_id: saved_project_id,
-        client_id: client.id,
-        date: entry_date,
-      },
-    )
     save!(validate: false)
   end
   alias save_in_progress save_in_progress!
 
   def save_not_in_progress!
-    transaction do
-      self.project_id = project_id || project.project_id
-      wip&.destroy!
-      save!
-    end
+    self.project_id = project.project_id
```

Comment:
maybe reuse this method from above?
```suggestion
    set_hud_project_id_from_project_pk
```

Reply:
set_hud_project_id_from_project_pk won't change project_id (wip) status so probably not quite right here